cmd/do-agent/config.go

```diff
@@ -8,7 +8,6 @@ import (
 	"strings"
 	"time"
 
-	"github.com/digitalocean/go-metadata"
```

Comment:
I guess you can probably remove this package from vendoring too

Reply:
good call